multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -159,45 +159,28 @@ coreos:
         ExecStartPre=/usr/bin/curl http://127.0.0.1:8080/version
         ExecStart=/opt/bin/install-kube-system
 
-{{ if .UseCalico }}
```

Comment:
We don't need a separate service for this. If the calico manifests are present in `/srv/kubernetes/manifests`, they get installed as part of `install-kube-system.service` now